spooky-oysters/React-FrontEnd
# Km training programs

```sh
git fetch --all
git checkout km-training-programs
npm start
```
In Browser, navigate to:
http://bangazon.com:3000/

Steps to test:
Should see Training Program with 3 buttons
1. List button should display lists
1. Clicking on any of the titles should display a Hello World Message
1. Clicking on the 'Add New' button should clear the list
1. Clicking on the 'Collapse' button should clear the list
